app/views/about/hiring_rails_developer.html.haml

```diff
@@ -0,0 +1,207 @@
+        Here is an idea of some of the things you'll work on through the first 12 months:
+      %ul.text-p-xlarge.mb-20
+        %li Working with the front-end dev to build our new automated mentoring management system.
+        %li Moving from our current mail provider to SES, writing a webhooks API to store underlying deliverability data (and saving us 90% costs on email!)
+        %li Developing out a prototype for our new Exercism Teams platform.
+        %li Adding a variety of action/reaction mechanisms to improve retention.
+        %li Working through many of the <a href="https://github.com/exercism/exercism/issues">open issues</a> that our community have opened.
+      %ul.text-p-xlarge.mb-16
+        You'll be working day-to-day with:
+      %ul.text-p-xlarge.mb-16
+        %li Our Head of Open Source, who manages our wider codebase (written in 30 languages) and its integration points into the primary backend.
+        %li Our front-end develop whose React consumes the backend API and presents it to our end-users.
+        %li Our CEO, who has written most of the existing Rails backend, and will continue to contribute features from time-to-time.
+      %p.text-p-xlarge.mb-40
+        We'll also encourage you to get involved in our wider community, to tap into their collective knowledge and experience, but this isn't compulsary.
+
+      %h2.text-h2.mb-12 Is this job right for you technically?
+
+      %p.text-p-xlarge.mb-16
+        The right person for this role will be someone who loves programming, is passionate about education, and who shares our values.
+        We're not worried about your qualifications - what matters to us is that you'll be able to do a great job of developing our backend.
+      %p.text-p-xlarge.mb-12
+        From a technical perspective, we're looking for someone who has:
+      %ul.text-p-xlarge.mb-16
+        %li
+          Strong Ruby and Rails skills.
+          You should have worked on product Rails codebases and solved meaningful problems in Rails.
+          You should "get" Rails, feel comfortable in it, and enjoy using it.
+          You should understand "why" things work as they do - for example do you "get" how Zeitwerk works?
+        %li A fundamental belief in the importance of writing good tests. We generally encourage using TDD where sensible. At Exercism we use minitest, not spec-based tests.
+        %li An interest in performant SQL (we use MySQL). We can help get you up to speed if you've not got much experience in this area, but we'd like you to care about it.
+        %li
+          Ideally some experience with AWS.
+          We use Terraform to build our infrastructure (<a href="https://github.com/exercism/terraform">code here</a>).
+          Experience here isn't essential, but it would be good to have someone who has an interest in learning this even if they haven't done much before.
```

Comment:
```suggestion
          Experience here isn't essential, but it would be good to have someone who has an interest in learning this, even if they haven't done much before.
```